Package.swift

```diff
@@ -451,7 +451,6 @@ let package = Package(
                 .target(name: "Apodini"),
                 .target(name: "ApodiniUtils"),
                 .target(name: "ApodiniVaporSupport"),
-                .target(name: "ApodiniOpenAPI"),
                 .target(name: "ApodiniDeployBuildSupport"),
                 .target(name: "ApodiniDeployRuntimeSupport"),
                 .product(name: "OpenAPIKit", package: "OpenAPIKit")
```

Comment:
I think we can remove this dependency here. Please check what dependencies are really needed for the deployment related packages.